pkg/config/config.go

```diff
@@ -0,0 +1,34 @@
+package config
+
+// Define constants for supported IDL types.
+const (
+	Unknown  string = ""
+	Thrift   string = "thrift"
+	Protobuf string = "protobuf"
+	// Reserved for Server Streaming
+)
+
+type Transport string
```

Comment:
defined but not used?

Reply:
It is used in
https://github.com/kitex-contrib/kitexcall/pull/1/files#:~:text=case%20config.-,Thrift,-%3A